src/core/Akka.Cluster/ClusterEvent.cs

```diff
@@ -1056,17 +1050,17 @@ private void SendCurrentClusterState(IActorRef receiver)
             receiver.Tell(state);
         }
 
-        private void Subscribe(IActorRef subscriber, ClusterEvent.SubscriptionInitialStateMode initMode, IEnumerable<Type> to)
+        private void Subscribe(IActorRef subscriber, ClusterEvent.SubscriptionInitialStateMode initMode, ImmutableHashSet<Type> to)
```

Comment:
Multiple enumeration issue again - the only input for this `private` method is an `ImmutableHashSet<Type>` so there's no point in taking a more generic argument.